robinson/ronbinson.cpp

```diff
@@ -0,0 +1,79 @@
+#include <fstream>
+
+using namespace std;
+
+ifstream input("robinson.in");
+ofstream output("robinson.out");
+
+int m, n, c, l;
+int a[21][21];
+
+void initializeFirstRowAndFirstColumn()
+{
+	input >> m >> n >> l >> c;
+
+	for (int i = 1; i <= m; i++)
+	{
+		a[i][1] = n + i - 1;
+		a[1][i] = n + i - 1;
```

Comment:
```suggestion
		a[1][i] = a[i][1];
```

you can write like that, because a[i][1] is already calculated and it is the same, like a mirror